tests/python/distributed/test_data_parallel.py

```diff
@@ -77,18 +79,28 @@ def test_data_parallel():
 )
 @with_seed(0)
 def test_zero_opt_1():
-    dctx = dist.get_context()
-    dctx.enable_data_parallel = True
-    dctx.zero_opt_level = 1
-    device = f"cuda({dctx.local_rank})"
+    dcfg = dist.get_config()
+    dcfg.enable_data_parallel = True
+    dcfg.zero_opt_level = 1
+    comm = dist.get_communicator()
+    device = f"cuda({comm.local_rank})"
 
     run_model(device)
 
-    dctx.enable_data_parallel = False
-    dctx.zero_opt_level = 0
+    dcfg.enable_data_parallel = False
+    dcfg.zero_opt_level = 0
 
 
 if __name__ == "__main__":
+    if os.environ.get("RAF_FILE_STORE_PATH", None):
+        dist.set_default_communicator("void")
+        comm = dist.get_communicator()
+        size = int(os.environ.get("OMPI_COMM_WORLD_SIZE"))
+        rank = int(os.environ.get("OMPI_COMM_WORLD_RANK"))
```

Comment:
Could we just set a fixed number. We are using 2 GPU to run the test. `OMPI_COMM_WORLD_SIZE` and `OMPI_COMM_WORLD_RANK` are for OpenMPI, MPICH2 will not work.